pkg/strategy/controller.go

```diff
@@ -113,7 +115,7 @@ func normalizeSparkDependencies(spec *v1.JaegerStorageSpec) {
 		spec.Dependencies.Enabled = &trueVar
 	}
 	if spec.Dependencies.Image == "" {
-		spec.Dependencies.Image = viper.GetString("jaeger-spark-dependencies-image")
+		spec.Dependencies.Image = fmt.Sprintf("%s:%s", viper.GetString("jaeger-spark-dependencies-image"), version.Get().Jaeger)
```

Comment:
This should not be set. Spark dependencies do not follow jaeger release version schema.